types/gorilla-engine/index.d.ts

```diff
@@ -412,14 +413,14 @@ declare namespace GorillaEngine {
         hint?: string;
         allowedExtensions?: string;
         defaultLocation?: string;
-    }): void;
+    }): Promise<any>;
```

Comment:
It looks to me like this function returns an array: <https://docs.gorilla-engine.com/Gorilla_Engine_JavaScript_API_Reference/Core%20Classes.html#gorillaengineopenfilechooseroptions--promiseltarrayltstringgtgt>

```suggestion
    }): Promise<string[]>;
```